admin/client/src/components/Preview/Preview.js

```diff
@@ -7,6 +7,19 @@ import SilverStripeComponent from 'lib/SilverStripeComponent';
  */
 class Preview extends SilverStripeComponent {
 
+  constructor(props) {
+    super(props);
+
+    this.handleBackClick = this.handleBackClick.bind(this);
+  }
+
+  handleBackClick(event) {
```

Comment:
If onBack isn't supplied, it will just have a no-op. I think we should throw an error if onBack isn't set and the button is clicked. 

Alternatively, if onBack isn't supplied, remove the back button altogether.